website/content/docs/job-specification/job.mdx

```diff
@@ -105,8 +105,10 @@ job "docs" {
   at fixed times, dates or intervals.
 
 - `priority` `(int: 50)` - Specifies the job priority which is used to
-  prioritize scheduling and access to resources. Must be between 1 and 100
-  inclusively, with a larger value corresponding to a higher priority.
+  prioritize scheduling and access to resources.
+  Must be between 1 and [`server` block `job_max_priority`] inclusively,
```

Comment:
I'd put this as "Must be between 1 and the server's `job_max_priority` configuration"